src/grpc.appup.src

```diff
@@ -1,6 +1,6 @@
-%% -*-: erlang -*-
-{"0.6.4",
- [{"0.6.3",
+%% -*- mode: erlang -*-
+{VSN,
```

Comment:
Don’t use the VSN variable. It is only assigned a value in our `emqx/relup-helper` plugin. In order to provide it for use for other users, we need to use a standard approach here.
```suggestion
{"0.6.5",
```